repos/system_upgrade/common/actors/removeupgradefailedflag/libraries/removeupgradefailedflag.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+from leapp.libraries.stdlib import api
+
+LEAPP_UPGRADE_FAILED_FLAG_FILE = '/var/log/leapp/.leapp_upgrade_failed'
```

Comment:
Sorry for late comment. We decided with @pirat89 we would rather put the file into the "/root/tmp_leapp_py3/". Thanks.

Reply:
Changed, squashed and tested.